core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/deploy/maven/AbstractMavenDeployer.java

```diff
@@ -84,6 +85,7 @@ public void merge(S source) {
         this.username = merge(this.username, source.getUsername());
         this.password = merge(this.password, source.getPassword());
         this.authorization = merge(this.authorization, source.getAuthorization());
+        this.mayDeploySnapshots = merge(this.mayDeploySnapshots, source.getMayDeploySnapshots());
```

Comment:
Display the value of this field in `asMap`
Subclasses `Nexus2MavenDeployer` and `MavenCentralMavenDeployer` must set the filed value to `true` in their constructor.

Reply:
I thought so. Thanks.